Documentation/ApiOverview/Workspaces/Index.rst

```diff
@@ -69,7 +69,7 @@ The most basic form of a preview is when a live record is selected and
 you lookup a future version of that record belonging to the current
 workspace of the logged in backend user. This is very easy as long as
 a record is selected based on its "uid" or "pid" fields which are not
-subject to versioning; You simply call :code:`sys_page->versionOL()` after
+subject to versioning: call :code:`sys_page->versionOL()` after
```

Comment:
The lowercase "c" in "call" is correct here:

"When a colon introduces a phrase or an incomplete sentence that is meant to add information to the sentence before it, do not capitalize the first word after the colon unless it is a proper noun."
https://www.grammarly.com/blog/capitalization-after-colons/